packages/api/src/brevo-contact/dto/brevo-contacts.args.ts

```diff
@@ -23,7 +23,6 @@ export class BrevoContactsArgsFactory {
 
             @Field(() => String, { nullable: true })
             @IsOptional()
-            @IsEmail()
```

Comment:
Why do you remove this?

Reply:
Because the email field for this query is filled by a search input. This causes an error when typing part of an email address. 

For example, I want to search for `test@example.com` but typing `t` already causes a validation error